spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/autoconfiguration/XsuaaAutoConfiguration.java

```diff
@@ -48,6 +50,20 @@ public XsuaaServiceConfiguration xsuaaServiceConfiguration() {
 		}
 	}
 
+	private static class PropertyConditions extends AllNestedConditions {
+
+		public PropertyConditions() {
+			super(ConfigurationPhase.PARSE_CONFIGURATION);
+		}
+
+		@ConditionalOnProperty(prefix = "spring.xsuaa", name = "multiple-bindings", havingValue = "false", matchIfMissing = true)
+		static class MultipleBindingsCondition { }
+
+		@ConditionalOnProperty(prefix = "spring.xsuaa", name = "disable-default-property-source", havingValue = "false", matchIfMissing = true)
```

Comment:
matchIfMissing must be false i guess

Reply:
The condition matches if the property is not set which means that the property source is **not** disabled (=enabled).
If it would be set to matchIfMissing = false then the property would explicitly be set to false to not disable the property source.

It works just like the _multiple-bindings_ property. If it is set to true then the default property source is disabled. In any other case it is enabled.

Reply:
Correct :-) I've checked the full test case. The default test case works without any changes!